modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheMapEntry.java

```diff
@@ -138,7 +139,9 @@
     /** */
     public static final GridCacheAtomicVersionComparator ATOMIC_VER_COMPARATOR = new GridCacheAtomicVersionComparator();
 
-    /** Property name for entry lock timeout. */
+    /** Property name for entry lock timeout in milliseconds. Default is 1000. */
```

Comment:
I would say "Sets the entry's lock timeout in milliseconds. Defaults to 1000 milliseconds"

Reply:
Fixed.